src/components/DeveloperUtility/DeveloperUtility.tsx

```diff
@@ -14,10 +14,7 @@ import NotesAdjustment from './NotesAdjustment';
 const DeveloperUtility = () => {
   const [isExpanded, setIsExpanded] = useState(false);
 
-  // only show the developer utilities if we're in development mode
-  if (process.env.NEXT_PUBLIC_VERCEL_ENV === 'production') {
-    return <></>;
-  }
+  return <></>;
```

Comment:
Any reason why we removed this?

Reply:
no, I just needed to see the design of ChaptherHeader, will put it back once the code is ready for review